java/Ice/greeter/client/src/main/java/com/zeroc/demos/greeter/Client.java

```diff
@@ -9,7 +9,7 @@
 
 class Client {
     public static void main(String[] args) {
-        // Create an Ice communicator to initialize the Ice runtime. The communicator is disposed before the program
```

Comment:
`disposed` in a C# specific term; In Java, objects are _destroyed_.

Reply:
See the comment in Server.java

Reply:
I updated the comment to be the same!
I thought the difference in wording was intentional...